app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -1,4 +1,4 @@
-<% OregonDigital::GenericMetadata::PROPERTIES.map(&:to_sym).each do |attr| %>
+<% OregonDigital::GenericMetadata::PROPERTIES.map(&:to_sym).reject{|attr| attr==:oembed_url}.each do |attr| %>
```

Comment:
I hope this doesn't sound nitpicky.. but since rubocop doesn't check views, it wouldn't point this out.. can you make the `reject` a little more readable

`.reject { |attr| attr == :oembed_url }`

Reply:
Not at all. Also, I forgot to check rubocop anyways.
Need to get more in the habit